src/core/src/core_logic/PatchInstaller.py

```diff
@@ -19,6 +19,7 @@
 import os
 import time
 from core.src.bootstrap.Constants import Constants
+import math
```

Comment:
nit: ordering of imports - alphabetical on common libs first, then alphabetical on imports from within the code base.

Reply:
Done